### ISSUE TITLE
Shall we delete the comment `fixme(#3553)`?

### ISSUE THREAD

**Issue by tigercosmos** [servo/servo#21495]
https://github.com/servo/servo/blob/a1abdde8c62d01a144d12ae4b425d6b062bad41b/components/script/dom/htmlformelement.rs#L495-L498

#3553 is closed.
There is a merged PR in #15938.

**Comment by tigercosmos** [comment]
cc @KiChjang

**Comment by jdm** [comment]
I don't think that code has been updated to take advantage of the changes in #3553 yet.

**Comment by tigercosmos** [comment]
I think we usually fill the issue number, which is opened for the description, into `FIXME(#<number>)`.

In this case, maybe modify this comment to avoid misleading is better.
And since the PR is already merged, it could also be: 
```
// FIXME: This is an incorrect way of getting controls owned by the 
//             form, should refactor this based on html5ever's form owner(#3553)
```

**Comment by KiChjang** [comment]
I think the better solution here is to fix the underlying issue, rather than to fix the wording of the comments. @tigercosmos Do you want to make a PR for the changes?

**Comment by tigercosmos** [comment]
I can have a try. I don't look much about this code. @KiChjang Do you remember why you said it is incorrect, and what do you expect it would be like?

**Comment by pshaughn** [comment]
At least one thing is testably wrong relating to this. html/semantics/forms/the-form-element/form-nameditem.html expects an earlier `<input form=a name=b>` to be picked up when a `<form id=a></form>` appearing later in the page is going through its controls, and it doesn't find it.

**Comment by pshaughn** [comment]
There's a FIXME(#3553) that probably really means this issue.
